sign_recognizer.py

```diff
@@ -128,42 +129,13 @@ def process_video(video_filepath, start_frame, end_frame):
     return batched_frames
 
 
-
-def load_inception_model(device=0):
-    """
-    Args:
-        device: int
-    Returns:
-        pretrained_i3d_model: InceptionI3d
-    """
-    
-    # Initialize model
-    pretrained_i3d_model = InceptionI3d(400, in_channels=3)
-
-    # Load the general inception model weights
-    pretrained_i3d_model.load_state_dict(torch.load(ID3_PRETRAINED_WEIGHTS_PATH))
-
-    # Adapt the final layer for the number of classes we expect
-    pretrained_i3d_model.replace_logits(NUM_CLASSES)
-
-    # Load the weights for the fine-tuned model on ASL
-    pretrained_i3d_model.load_state_dict(torch.load(WLASL_PRETRAINED_WEIGHTS_PATH, map_location=torch.device('cpu')))
-
-    # Move to GPU
-    # i3d.cuda(device=device)
-
-    # Add data parallelism layer (but this is not actually that useful here since we're only using 1 example)
-    pretrained_i3d_model = nn.DataParallel(pretrained_i3d_model)
-
-    # Put model in inference mode
-    pretrained_i3d_model.eval()
-
-    # pretrained_model = PL_resnet50.load_from_checkpoint(checkpoint_path=weight_loc).eval()#.cuda(device=0)
-    return pretrained_i3d_model
-
-
 def main():
     parser = argparse.ArgumentParser(description=__doc__.split("\n")[0])
+    # parser.add_argument(
+    #     "model_path",
```

Comment:
I think we should still give the user a potential location of the torchscript model, and if they don't provide one, then we use the default one... I think that's how it's implemented in the FSDL lab too